packages/frontend/src/components/project/ArchivedBar.tsx

```diff
@@ -3,23 +3,24 @@ import React from 'react'
 
 import { ArchivedIcon } from '../icons/symbols/ArchivedIcon'
 
-export interface ArchivedBarProps {
-  text: string
-}
-
-export function ArchivedBar({ text }: ArchivedBarProps) {
+export function ArchivedBar() {
   return (
     <div
       className={cx(
-        'mb-6 flex justify-center p-2.5 md:px-0',
-        'rounded-lg bg-gray-800',
+        'mb-6 flex justify-center py-2.5 md:px-0',
```

Comment:
I'm not sure what's your goal with `md:px-0` (px is 0 already, as it's never set to a value).